src/test/java/edu/unc/lib/boxc/migration/cdm/services/SourceFileServiceTest.java

```diff
@@ -47,13 +48,17 @@ public class SourceFileServiceTest {
 
     private Path basePath;
 
-    @Before
+    @BeforeEach
     public void setup() throws Exception {
         project = MigrationProjectFactory.createMigrationProject(
-                tmpFolder.getRoot().toPath(), PROJECT_NAME, null, "user", CdmEnvironmentHelper.DEFAULT_ENV_ID);
+                tmpFolder, PROJECT_NAME, null, "user", CdmEnvironmentHelper.DEFAULT_ENV_ID);
         Files.createDirectories(project.getExportPath());
 
-        basePath = tmpFolder.newFolder().toPath();
+        File testFolder = new File(String.valueOf(tmpFolder), "testFolder");
+        if (!testFolder.exists()){
+            testFolder.mkdirs();
+        }
+        basePath = testFolder.toPath();
```

Comment:
It seems like testFolder shouldn't ever exist at this point, since tmpFolder will have just been created. We should also generally favor java.nio clients over java.io (so Path versus File), unless there's a good reason to use File. So I think we might be able to change this section to:
```suggestion
        basePath = tmpFolder.resolve("testFolder");
        Files.createDirectory(basePath);
```